src/app.js

```diff
@@ -1,13 +1,142 @@
 // Vendor Modules
 import $ from 'jquery';
+import 'jquery-modal'
 import _ from 'underscore';
 
 // CSS
 import './css/foundation.css';
 import './css/style.css';
 
-console.log('it loaded!');
+import Trip from './app/models/trip';
+import TripList from './app/collections/trip_list';
+import Reservation from './app/models/reservation';
+
+
+const tripList = new TripList();
+const render = function(tripList) {
+  const tripTemplate = _.template($('#trip-template').html());
+  const $tripList = $('#trip-list');
+  $tripList.empty();
+  tripList.forEach((trip) => {
+    $tripList.append($(tripTemplate(trip.attributes)).attr('id', `${trip.id}`));
+  })
+}
+
+const reservationFields = ['name', 'age', 'email']
+const tripFields = ['name', 'continent', 'about', 'weeks', 'cost', 'category'];
+const events = {
+  addTrip(event) {
+    console.log("I want to make a new trip");
+    event.preventDefault();
+    const tripData = {};
+    tripFields.forEach((field) => {
+      const value = $(`#newTripForm input[name=${field}]`).val();
+      if (value !== "") {
+        tripData[field] = value
+      }
+    });
+
+    const trip = new Trip(tripData);
+
+    if (trip.isValid()){
+      trip.save({}, {
+        success: events.successfulSave,
+        error: events.failedSave,
+      });
+    } else {
+      for (let key in trip.validationError) {
+        $('#status-messages ul').append(`<li>${key}:  ${trip.validationError[key]}</li>`);
+        $('#status-messages').show();
+      }
+    }
+  },
+
+  sortTrips(event) {
+    $('.sort').removeClass('current-sort-field');
+    const classes = $(this).attr('class').split(" ");
+    $(this).addClass('current-sort-field');
+    if (tripList.comparator === classes[1]) {
```

Comment:
I would try not to hardcode which index to pull an attribute from, since it's not guaranteed that it will be the right class